src/Runtime/Runtime/Core/Rendering/INTERNAL_HtmlDomManager.cs

```diff
@@ -656,14 +647,22 @@ public static void SetDomElementStyleProperty(object domElementRef, List<string>
             else
             {
                 string uniqueIdentifier = ((INTERNAL_HtmlDomElementReference)domElementRef).UniqueIdentifier;
-                string settingProperties = string.Empty;
+                string javaScriptCodeToExecute;
                 var value = ConvertToStringToUseInJavaScriptCode(attributeValue);
-                foreach (string propertyName in propertyCSSNames)
+                if (propertyCSSNames.Count == 1)
                 {
-                    settingProperties += $"element.style.{propertyName} = {value};";
+                    javaScriptCodeToExecute = $@"document.setDomStyle(""{uniqueIdentifier}"", ""{propertyCSSNames[0]}"", {value})";
+                }
+                else
+                {
+                    string settingProperties = string.Empty;
+                    foreach (string propertyName in propertyCSSNames)
+                    {
+                        settingProperties += $"element.style.{propertyName} = {value};";
+                    }
+                    javaScriptCodeToExecute =
+                        $@"var element = document.getElementByIdSafe(""{uniqueIdentifier}"");if (element) {{ {settingProperties} }};";

```

Comment:
Do not use 'getElementByIdSafe' if the element is null checked.
```suggestion
                        $@"var element = document.getElementById(""{uniqueIdentifier}"");if (element) {{ {settingProperties} }};";
```